build/make/self-update.mk

```diff
@@ -1,5 +1,7 @@
+##@ Makefile management
+
 .PHONY: update-makefiles
-update-makefiles: do-update-makefiles
+update-makefiles: do-update-makefiles ## Update Makefiles to MAKEFILES_VERSION
```

Comment:
Maybe we should keep the descriptions as small as possible in order to avoid information overload towards the user. I suggest to go with `Update Makefiles`

Reply:
The problem is that nothing happens if the user just runs this target. The MAKEFILES_VERSION has to be set to the desired version before, and I want to give a hint to this here.

Reply:
Ok